src/style.css

```diff
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+input {
+  outline: none;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+
+  --primary-color: #e1e7ed;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  padding: 20px 8px;
+}
+
+.form {
+  position: relative;
+}
+
+.form:not(:last-child) {
```

Comment:
It's better to set `margin-bottom` to all elements where it's needed (e.g. for `.label`)